pom.xml

```diff
@@ -144,20 +144,14 @@
         <dependency>
```

Comment:
@sBouzols can we remove the whole dependency powsybl-dynawo-commons ? This seems to have been introduced in https://github.com/gridsuite/dynamic-simulation-server/pull/73/files

Reply:
removed